src/components/AQICard/styles.css

```diff
@@ -0,0 +1,73 @@
+.info-container {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 20px 0;
+  gap: 12px;
+  box-sizing: border-box;
+}
+
+.aqi-container {
+  width: auto;
+  height: auto;
+  background-color: #ffffff43;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 6px;
+  padding: 8px;
+  border-radius: 12px;
+}
+
+.aqi-box {
+  width: 40px;
+  height: 40px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 2rem;
+  font-weight: 600;
+  background-color: #000;
+}
+
+.aqi-indicator {
+  width: auto;
+  height: auto;
+  /* background-color: #ffffff43; */
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 6px;
+  padding: 0 8px;
+  border-radius: 12px;
+}
+
+.aqi-indicator-box {
+  width: 40px;
+  height: 40px;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: transparent;
+  gap: 12px;
+}
+
+.aqi-indicator-circle {
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background-color: #fff;
+}
+
+.aqi-indicator-text {
```

Comment:
can you also give background for text so when the background of the app changes, the text is still visible!

![image](https://user-images.githubusercontent.com/105518066/227313011-ee3903a7-4720-40d0-b04a-6004f17364ae.png)